omero/developers/Web/WebGateway.txt

```diff
@@ -136,7 +136,7 @@ Individual parameters are:
        ?c=3         # only Channel 3 is active
        ?c=3,4       # Channels 3 and 4 are active
 
--  **Channel colour** e.g. to set the colours for channels 1 to red and 2
+-  **Channel color** e.g. to set the colors for channels 1 to red and 2
    to green and 3 to blue:
 
    ::
```

Comment:
``Rendering levels e.g. to set the cut-in and cut-out values for a 3 Channel image.``
replace by image with 3 channels to match the rest